# PD_stratified_GWAS

In this study

1) We used IPDGC data to run stratified GWAS for carriers of H1/H1 and H2 MAPT haplotypes, scirpt attached
2) Interaction analysis was done using plink and TLTO (Two-loci to OR) from VARI3 package (https://github.com/alexcis95/VARI3).
3) Miami plots representing GWASs for carriers and non-carriers were created using the Hudson package in R (https://github.com/anastasia-lucas/hudson)
4) Locus zoom plots for regions of interest have been made using the online tool http://locuszoom.org/.
5) Beta-beta plots were created using custom script from https://github.com/lynnekrohn/RBD_GWAS/blob/main/3_Synucleinopathy_beta-beta.md
6) Buden analysis for rare variants and meta-analysis was done using SKAT-O package (https://github.com/leelabsg/SKAT) as we previosly described (https://github.com/gan-orlab/ARSA)

